src/main/java/org/terasology/anatomy/AnatomySkeleton/SkeletalSystem.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.anatomy.AnatomySkeleton;
+
+import com.google.common.collect.Lists;
+import org.terasology.anatomy.AnatomySkeleton.component.BoneComponent;
+import org.terasology.anatomy.AnatomySkeleton.component.BrokenBoneComponent;
+import org.terasology.anatomy.AnatomySkeleton.event.BoneHealthChangedEvent;
+import org.terasology.anatomy.component.AnatomyComponent;
+import org.terasology.anatomy.component.PartSkeletalDetails;
+import org.terasology.anatomy.event.AnatomyEffectAddedEvent;
+import org.terasology.anatomy.event.AnatomyEffectRemovedEvent;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.logic.console.commandSystem.annotations.Command;
+import org.terasology.logic.console.commandSystem.annotations.Sender;
+import org.terasology.network.ClientComponent;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A skeletal system which works with Anatomy. Provides a basic bone breaking effect with 3 levels of severity.
+ */
+@RegisterSystem
+public class SkeletalSystem extends BaseComponentSystem {
```

Comment:
Maybe some name ambiguity here vs `SkeletalAuthoritySystem` ? Since this system is annotated with plain `@RegisterSystem` it'll act as *both* authority and client. Should the authority-specific methods be moved to the authority system and this maybe renamed to work as a Client-side `@Command` centric system?